profiles/multisite_drupal_communities/modules/custom/og_content_type_admin/og_content_type_admin.module

```diff
@@ -154,7 +142,9 @@ function og_content_type_admin_og_create_links($group) {
   if ($group) {
     if (_og_content_type_admin_is_admin($group)) {
       $links[] = l(t('Manage group content types'), "node/$group->nid/ogct", array(
-        'title' => t('Lets you decide which content types your group users can create.')));
+        'title' => t('Lets you decide which content types your group users can create.'),
+        )
+      );
```

Comment:
Something doesn't look right here with the indentation. It's better not to declare arrays inline in a function call, just separate out the arguments onto different lines:

```
$text = t('Manage group content types');
$path = "node/$group->nid/ogct";
$attributes = array('title' => ...);
$links[] = l($text, $path, $attributes);
```